VocaDb.ResXFileCodeGenerator/AnalyzerReleases.Unshipped.md

```diff
@@ -0,0 +1,9 @@
+; Unshipped analyzer release
+; https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md
+
+### New Rules
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+VocaDbResXFileCodeGenerator001 | ResXFileCodeGenerator | Warning | StringBuilderGenerator
+VocaDbResXFileCodeGenerator002 | ResXFileCodeGenerator | Warning | StringBuilderGenerator
+VocaDbResXFileCodeGenerator003 | ResXFileCodeGenerator | Error | StringBuilderGenerator
```

Comment:
I don't quite understand the purpose of these two files `AnalyzerReleases.Unshipped.md` and `AnalyzerReleases.Shipped.md`. Should I move these rules (adding `## Release 3.1` before `### New Rules`) to `AnalyzerReleases.Shipped.md` *after* publishing a nuget package?

Reply:
No. I just used the built-in quick fix which made those files. Don't have a clear idea what they actually do.

Reply:
"Don't have a clear idea what they actually do." -  Me too.

What I found is https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md but is it helpful? No